flixel/FlxSprite.hx

```diff
@@ -1357,6 +1358,30 @@ class FlxSprite extends FlxObject
 	{
 		return antialiasing = value;
 	}
+	
+	private inline function checkFlipX():Bool
+	{
+		var doFlipX:Bool = FlxMath.boolXOR(flipX, _frame.flipX);
```

Comment:
There's not really much of a reason for this line to be always executed - it's just the else case of `if (animation.curAnim != null)`.